### PR TITLE
navigation rail optimization

### LINKED ISSUE #860: FeatureInfo panel should be restoreable
After closing the main menu in **portrait mode**. the FeatureInfo panel can not be opened again
1) close the main menu containing the FeatureInfo
grafik
2) no chance to re-open it
grafik

### PR DESCRIPTION
**Pull request by alsturm**
also Fixes #860

### CODE REVIEW

**Review by thiloSchlemmer — DISMISSED** [review]
Thx

File: src/modules/menu/components/navigationRail/NavigationRail.js [hunk 1]
```diff
@@ -104,18 +97,19 @@ export class NavigationRail extends MvuElement {
 		};
 
 		const openTab = (tabId) => {
+			setCurrentTool(null);
 			setTab(tabId);
-			open();
+			isPortrait ? toggle() : open();
```
Comment (thiloSchlemmer) [inline src/modules/menu/components/navigationRail/NavigationRail.js:102]: Branch not covered by test

Comment (alsturm) [reply src/modules/menu/components/navigationRail/NavigationRail.js:102]: done

**Comment by taulinger** [comment]
Will check this PR tomorrow...

**Review by taulinger — APPROVED** [review]

**Pull request merged**